### ISSUE TITLE
Static Files

### ISSUE THREAD

**Issue by mpk3** [mpk3/DjangoWebApps#14]
Static files are currently spread throughout multiple apps in the project. I think the collecticstatic command is meant to to put everything in one place but currently my templates are in there own separate folder for each app individually. This is leading to a lot of duplicate code. I noticed for cloud deployment there are libraries like django-storage. This is overkill for the current application but it leads me to believe I am not using Django's bultin static features correctly.

TLDR: Static files are spread out in multiple applications. Have not used Django collectstatic correctly(?)

- [ ] Figure out what the proper methodology/project structure is for handling this.
